homeassistant/components/motionmount/entity.py

```diff
@@ -0,0 +1,26 @@
+"""Support for MotionMount sensors."""
+from homeassistant.helpers.device_registry import DeviceInfo, format_mac
+from homeassistant.helpers.entity import Entity
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN
+from .coordinator import MotionMountCoordinator
+
+
+class MotionMountEntity(CoordinatorEntity[MotionMountCoordinator], Entity):
+    """Representation of a MotionMount entity."""
+
+    _attr_has_entity_name = True
+
+    def __init__(self, coordinator: MotionMountCoordinator) -> None:
+        """Pass coordinator to CoordinatorEntity."""
+        super().__init__(coordinator)
```

Comment:
You could probably set always_update=False here since data can be compared with __eq__. That way you won't generate state writes when nothing has changed

Reply:
If only one of the entries in the dictionary changes all entities will be notified of an update. Is there also a way to prevent the non-changed entities from writing their state to HA? Will HA filter out duplicate updates from entities?